DigitalLearningSolutions.Data/Services/RegistrationService.cs

```diff
@@ -136,7 +136,7 @@ public void RegisterCentreManager(AdminRegistrationModel registrationModel, int
 
             CreateDelegateAccountForAdmin(registrationModel, jobGroupId);
 
-            registrationDataService.RegisterAdmin(registrationModel);
+            registrationDataService.RegisterAdmin(registrationModel, true);

```

Comment:
Is it intended that this value not actually be passed through from anywhere but just assumed to be true if this location in the code is reached? That seems a strange way to do it but I can be convinced.

Reply:
I've pushed the value up to the service method parameters so that it matches what I did for UAR.
I don't think we should move the clockService call just so that it more closely matches UAR and will cause less confusion when that gets merged.